### DISCUSSION TITLE
Rust executable using slint is crashing on some Windows computers

### DISCUSSION THREAD

**Discussion by Sulquendi** [slint-ui/slint#6436] [Q&A]
Hello !

I've just got informed that my exe using slint to display a simple splash screen is crashing on some Windows computers with ACCESS DENIED error (Exception code: 0xC0000005). The faulting module mentioned is atio6axx.dll (which appears to be an AMD OpenGL lib)

My first attempt was to update my project : switched to rust 1.81 (was 1.76) and to slint 1.8.0 (was 1.4.1) but still got the error. 

Any idea about the directions to take to diagnose and fix this stuff ?

Thanks for your help !

Regards

**Comment by ogoffart** [comment]
You can also try to enable the skia renderer by enabling the corresponding feature in Cargo.toml
( https://docs.slint.dev/1.8.0/docs/slint/src/advanced/backends_and_renderers#skia-renderer )
Maybe that's a bug in the GL driver and the way femtovg (the default backend) issue GL commands.
You can also try the software renderer.

**Comment by tronical** [comment]
Does it work if you remove `atio6axx.dll`?

I don't quite understand why glutin tries that dll on windows when looking for egl support. I might try to petition the removal.

Did you enable the `renderer-skia-opengl` feature by chance? Does it work if you remove it and just use `renderer-femtovg`?

**Comment by Sulquendi** [comment]
Thanks for your quick answer ! I now have one executable per renderer (skia, femtovg and software) , all are using winit backend, and we will test which one is working on a computer having the issue.

Best regards
